elements/cv_scaler.cc

```diff
@@ -196,11 +203,19 @@ void CvScaler::Read(
     note_ += 0.1f * interval;
   }
 
-  float modulation = pot_lp_[POT_RESONATOR_FM_ATTENUVERTER] * 49.5f *
+  // manually read the values of the exciter blow meta cv and input
+  // attenuverter to tranform the value of the FM attenuverter pot
+  float pot_fm = pot_lp_[POT_RESONATOR_FM_ATTENUVERTER];
+  float cv_fm = calibration_settings_.offset[CV_ADC_EXCITER_BLOW_META] -
+      cv_.float_value(CV_ADC_EXCITER_BLOW_META);
+  float value_fm = pot_fm + pot_lp_[POT_EXCITER_BLOW_META_ATTENUVERTER] * cv_fm;
+  CONSTRAIN(value_fm, 0.0f, 0.9995f);
+
```

Comment:
I did not include any sort of scale coefficient like your BIND macro did...I'm assuming I could do this to change the CV's effect amount on the FM input attenuation, but I believe it's probably okay that I didn't?  I'm curious if there's either a way to know how much you'd want to scale...mathematically figuring it out or some sort of running of tests?